lib/streamlit/elements/select_slider.py

```diff
@@ -0,0 +1,114 @@
+from streamlit.proto.Slider_pb2 import Slider as SliderProto
+from streamlit.errors import StreamlitAPIException
+from streamlit.type_util import ensure_iterable
+from .utils import _get_widget_ui_value
+
+
+class SelectSliderMixin:
+    def select_slider(
+        dg, label, options=[], value=None, format_func=str, key=None,
+    ):
+        """
+        Display a slider widget to select items from a list.
+
+        This also allows you to render a range slider by passing a two-element
+        tuple or list as the `value`.
+
+        The difference between `st.select_slider` and `st.slider` is that
+        `select_slider` accepts any datatype and takes a list as input, while
```

Comment:
In line above, reference that you can use a `tuple or list`, but this line says 'takes a list as input'

Reply:
Ah. Yes. I can add clarity. How about `and takes an iterable set of options`?